api/v1alpha1/openstack_ansibleee_types.go

```diff
@@ -134,12 +140,19 @@ type Role struct {
 	// +kubebuilder:default:="{{ primary_role_name | default([]) }}:{{ deploy_target_host | default('overcloud') }}"
 	Hosts string `json:"hosts,omitempty"`
 	// +kubebuilder:default:=free
+	// strategy defaults to free
```

Comment:
we should actually change the default to be linear. but that is unrelated to this PR

Reply:
Opened a PR for that here: https://github.com/openstack-k8s-operators/openstack-ansibleee-operator/pull/108